public/index.css

```diff
@@ -582,39 +582,46 @@
   /* ========================================
   *  Thank You Page (Mailing List Signup & Socail Share)
   */
-  #white-box {
-    width: 100%;
-    background: #ffffff;
-    padding: 1.5rem 2rem 3.5rem 2rem;
-    margin: 2rem auto 10rem auto;
-  }
-  @media screen and (min-width: 768px) {
-    #white-box {
-      width: 42rem;
-      padding: 2rem 3.5rem 4rem 3.5rem;
-    }
-  }
-  .share-btn {
-    display: block;
-    padding: 1.5rem;
-    text-align: center;
-    color: #ffffff;
-  }
-  .share-btn p {
-    margin: 0;
-  }
-  #facebook-share {
-    background: #4e7ac0;
-  }
-  #facebook-share:hover {
-    background: #5680c3;
-  }
-  #twitter-share {
-    background: #5ecdf3;
-  }
-  #twitter-share:hover {
-    background: #67d0f4;
-  }
+.share-page .container {
+  margin: 0px auto;
+  width: 420px;
+  overflow-x: hidden;
+  padding: 30px;
+  background: #FFF none repeat scroll 0% 0%;
+}
+
+.share-page h2 {
+  width: 620px;
+  margin: 0 auto;
+  padding: 30px 0;
+}
+
+ .share-page #facebook,
+.share-page #twitter {
+  text-align: center;
+}
+.share-page #facebook a,
+.share-page #twitter a {
+  color: #ffffff;
+  display: block;
+  padding: 20px;
+}
+.share-page #facebook,
+.share-page #twitter {
+  transition: background-color 200ms ease;
+}
+.share-page #facebook {
+  background-color: #4E7AC0;
+}
+.share-page #facebook:hover {
+  background-color: #406db4;
+}
+.share-page #twitter {
+  background-color: #5ECDF3;
+}
+.share-page #twitter:hover {
+  background-color: #46c6f1;
+}
```

Comment:
it wasn't indentation fixes actually. It was class name changes and add namespace for id/class